cmd/gen-go-sample/inittree.go

```diff
@@ -511,3 +515,21 @@ func snakeToCamel(s string) string {
 	}
 	return sb.String()
 }
+
+func printCommentLines(w *bufio.Writer, comment string, ind int) {
+	if comment == "" {
+		return
+	}
+	for _, c := range strings.Split(strings.TrimRight(comment, "\n"), "\n") {
+		indent(w, ind)
```

Comment:
How about simply
```
w.WriteString(fmt.Sprintf("%s// %c\n", indent(ind), c))
```

(with `indent` returning a `string`?)

Reply:
Same as below: `indent` is also used elsewhere. It does make the code shorter, but it will cause too much changes.